recipe/meta.yaml

```
@@ -56,15 +54,17 @@ test:
     - pyarrow.compute
```

```suggestion
    - pyarrow.acero
    - pyarrow.compute
```

Should this be added since the option for acero is set to on [here](https://github.com/AnacondaRecipes/pyarrow-feedstock/pull/19/files#diff-54cf74e113dd3f6d11e092fdb1d888ec82c69bdafbb15cfb6570c83ecad28f33R6)?

---

> Same thing with `pyarrow.dataset_parquet`?

---

And `pyarrow.parquet_encryption`?

---

And `pyarrow.substrait`?

---

addressed, thank you.